example/components/SlidingTabNavigationExample.js

```diff
@@ -17,6 +19,18 @@ export default class SlidingTabNavigationExample extends Component {
     },
   }
 
+  _gotToFirstTab = () => {
```

Comment:
small typo: `_goToFirstTab`

I'm thinking maybe better to have more abstract function:
`const _goToTab = (navigatorUID, tabId) => {/* ... */}`